CMakeLists.txt

```diff
@@ -437,7 +437,7 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
 	install(CODE "execute_process(COMMAND cp ${CMAKE_SOURCE_DIR}/../install-root/${_qt5Core_install_prefix}/plugins/geoservices/libqtgeoservices_googlemaps.dylib ${PLUGINDIR}/geoservices ERROR_QUIET)")
 	# this is a hack - but I don't know how else to find the macdeployqt program if it's not in the PATH
 	string(REPLACE moc macdeployqt MACDEPLOYQT ${QT_MOC_EXECUTABLE})
-	install(CODE "execute_process(COMMAND ${MACDEPLOYQT} ${APP_BUNDLE_DIR} ${MACDEPLOY_ARGS})")
+	install(CODE "execute_process(COMMAND ${MACDEPLOYQT} ${APP_BUNDLE_DIR} -no-strip ${MACDEPLOY_ARGS})")
```

Comment:
question: what's your thought on stripping release builds, though?
in other words, should this change be conditional to this not being a release build?
I'm not sure what the right answer is - can you remind me of the size difference?

Reply:
Desktop difference is approx. 200k (500k to 700k) I do not have the numbers on iOS but expect the difference in percentage to be the same.

I understood you made release build through a different route, but if you use this script it should depend on debug. I will update the patch later.

I assume your business trip means the upload feature will not be merged until you come back ?

Reply:
200k doesn't seem worth the inconvenience, given the overall size of the full app including libraries of tens of megabyte.
I'll merge as is.